src/rpc/mempool.cpp

```diff
@@ -463,6 +674,8 @@ void RegisterMempoolRPCCommands(CRPCTable& t)
     static const CRPCCommand commands[]{
         // category     actor (function)
         // --------     ----------------
+        {"rawtransactions", &sendrawtransaction},
+        {"rawtransactions", &testmempoolaccept},
```

Comment:
I am expecting that everything will move to a new `mempool` category later, so I am leaving as-is for now.

Reply:
I think that's acceptable reasoning.
Other than this, I can't find any blockers for this PR, and hence I think it can be merged.

ACK fac5a51c47fba21678fb35805e40d00fe7c891a0